global.d.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+declare namespace PolymerDecorators {
+  function customElement(tagname?: string): (clazz: any) => void;
```

Comment:
clazz is a class right? Maybe its type can be narrowed down a bit more from any?

Reply:
We could possibly do:

```ts
type CustomElementConstructor = {new(): HTMLElement};
function customElement(tagname?: string): (clazz: CustomElementConstructor) => void;
```

Reply:
Added TODO for now.